src/components/common/ViewerSidebar.jsx

```diff
@@ -30,6 +30,7 @@ import { BackendAPI } from '../common/API';
 import Metadata from '../common/Metadata';
 import FileDownloader from '../../js/http';
 import CollaborativeAnnotation from './CollaborativeAnnotation';
+import GProfilerModal from './GProfilerModal.tsx';
```

Comment:
```suggestion
import GProfilerModal from './GProfilerModal';
```
I would be happy to see this in `components/GProfiler/....`